test/views/html/TaxCodeDetailsViewSpec.scala

```diff
@@ -49,25 +49,25 @@ class TaxCodeDetailsViewSpec extends TaiViewSpec {
         messages("return.to.your.income.tax.summary"))
     }
 
-    "display an list of tax-code-part descriptions, for each tax code provided in the view model" in {
-      doc must haveUnorderedListWithId("taxCodeList1")
+    "display tax-code-part descriptions, for each tax code provided in the view model" in {
+      doc must haveDivWithId("taxCodeList1")
```

Comment:
is this ok? Changing from a unordered list to a div?

Reply:
we followed the prototype for this where div is used rather than a list, using a list meant custom css changes whereas with div it wasn't necessary